airflow/providers_manager.py

```diff
@@ -140,10 +141,21 @@ def _sanity_check(provider_package: str, class_name: str) -> bool:
         return False
     try:
         import_string(class_name)
+    except AirflowOptionalProviderFeatureException as e:
+        # When the provider class raises AirflowOptionalProviderFeatureException
+        # this is an expected case when only some classes in provider are
+        # available. We just log debug level here
+        log.debug(
+            "Exception when importing '%s' from '%s' package: %s",
```

Comment:
This sounds much to generic to me. Maybe something like _optional feature [module] from [provider] disabled due to [exception]_ instead.

It may also e worthwhile to add `exc_info=True`.

Reply:
Good point.

Reply:
I was also thinking if not to make it INFO level, but it will continue flooding the log. I wish we had "log once per app run" working acrross multiple processes and gunicorn restarts as well.